org.adempiere.ui.zk/WEB-INF/web.xml

```diff
@@ -8,6 +8,18 @@
 		<param-name>org.eclipse.jetty.servlet.SessionCookie</param-name>
 		<param-value>WEBUI_SESSIONID</param-value>
 	</context-param>
+
+	<filter>
+		<display-name>iDempiere WebUI Filter</display-name>
+		<filter-name>idempiereWebuiFilter</filter-name>
```

Comment:
please use SSO WebUI Filter for display name and filter name

Reply:
Done